ui/StatusQ/include/StatusQ/stringutilsinternal.h

```diff
@@ -11,9 +11,12 @@ class StringUtilsInternal : public QObject
     Q_OBJECT
 
 public:
-    explicit StringUtilsInternal(QObject* parent = nullptr);
+    explicit StringUtilsInternal(QQmlEngine* engine, QObject* parent = nullptr);
```

Comment:
```suggestion
    explicit StringUtilsInternal(QQmlEngine *engine, QObject *parent = nullptr);
```
I know it was formatted wrong before, but it's nice chance to adjust it to general convention :)

Reply:
We have this magical file https://github.com/status-im/status-desktop/blob/master/.clang-format which defines the formatting; and QtCreator obeys that when you touch the file and save it :)